.vsts-ci/sign-module-files.xml

```diff
@@ -10,5 +10,7 @@
 
     <file src="__INPATHROOT__\Microsoft.PowerShell.PSReadLine2.dll" signType="AuthenticodeFormer" dest="__OUTPATHROOT__\Microsoft.PowerShell.PSReadLine2.dll" />
     <file src="__INPATHROOT__\System.Runtime.InteropServices.RuntimeInformation.dll" signType="AuthenticodeFormer" dest="__OUTPATHROOT__\System.Runtime.InteropServices.RuntimeInformation.dll" />
```

Comment:
Does this come from .NET? Shouldn't this be already signed.

Reply:
Yes, this comes from .NET, and yes it's already signed. I will skip this one from the signing.